incubator/jaeger/templates/query-deploy.yaml

```diff
@@ -41,8 +41,21 @@ spec:
 {{ toYaml .Values.query.resources | indent 10 }}
         command:
         - "/go/bin/query-linux"
+{{ if ne .Values.elasticsearch.config.host "" }}
```

Comment:
Same as above: it's probably better to use a configmap.

Reply:
same as above

Reply:
`{{ if ne .Values.elasticsearch.config.host "" }}` could be written as `{{ if not .Values.elasticsearch.config.host }}`

Reply:
@unguiculus when I rewrote the if statement as suggested, the chart is actually incorrectly going through the first if statement. I also tried commenting out the `host: ""` key in the values.yaml, but it is still going through as if host were being set.

Reply:
Leave it for now. Maybe I was wrong. Need to check it.

Reply:
thanks. yea, I have been scratching my head... it should work, but i can't figure out why not.
```
[dev@k8controller ~]$ helm version
Client: &version.Version{SemVer:"v2.7.0", GitCommit:"08c1144f5eb3e3b636d9775617287cc26e53dba4", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.7.0", GitCommit:"08c1144f5eb3e3b636d9775617287cc26e53dba4", GitTreeState:"clean"}
```